cmd/pj-rehearse/main.go

```diff
@@ -19,6 +21,9 @@ import (
 	"github.com/openshift/ci-operator-prowgen/pkg/rehearse"
 )
 
+const masterConfigPath = "master/cluster/ci/config/prow/config.yaml"
+const masterJobConfigPath = "master/ci-operator/jobs/"
```

Comment:
One of my comments was not addressed, and I'm not sure whether it's an omission or intentional (because it was not explicity refused), so I'll reiterate.

I don't think these constants are necessary and even desirable - we have the paths relative to `openshift/release` repo available already, in the `pkg/diffs` package. They are not exported, but that can change because the paths are useful outside. The only difference is the `master/` prefix, present because that is, accidentally, where `createMasterWorktree` creates the worktree. This means these constants are tightly coupled to `createMasterWorktree`, which is not desirable - the coupling is unclear, at minimum. 

We can easily get rid of these constants if `createMasterWorktree` returns a path where it created the worktree, which is IMO a reasonable API. Such returned path can then be combined with the constants from the `pkg/diffs` package.

Reply:
I misunderstood your previous [comment](https://github.com/openshift/ci-operator-prowgen/pull/56#discussion_r249768324). Changes addressed in the latest push.